assets/overrides/guppy/oppia_symbols.json

```diff
@@ -1,31 +1,36 @@
 {
+    "_version": "2.0.0-alpha.2",
+    "_name": "base",
     "frac": {
         "output": {
             "latex": "\\frac{{$1}}{{$2}}",
             "small_latex": "\\frac{{$1}}{{$2}}",
             "text": "({$1})/({$2})"
         },
-        "current": 0,
-        "type": "fraction",
-        "attrs": {
-            "up": [1, 1],
-            "down": [2, 2],
-            "name": ["numerator", "denominator"]
-        }
+        "input":1,
+        "keys":["/"],
+        "attrs":{
+        "type":"fraction",
```

Comment:
Lint: follow styling of other dicts please. E.g. break after "{", indent, put "}" on a new line, etc.